### DISCUSSION TITLE
Solved: ValueError: 'binary_classification' is not a valid ModelMode

### DISCUSSION THREAD

**Discussion by yousha806** [pyg-team/pytorch_geometric#6381] [Q&A] [answered]
<img width="983" alt="Screenshot 2023-01-10 at 1 05 51 PM" src="https://user-images.githubusercontent.com/40205524/211490122-d3ab8ad4-26ce-45bb-ae9e-1adee22c49e7.png">

Hi I've been getting the above error, despite following the documentation for PyG for Graph Explanation. 

I have been this running this in Google Colab, and have the latest pytorch version. 

Would appreciate any help!

UPDATE: Solved by changing it simply to 'classification'

**Comment by rusty1s** [comment, marked as answer]
We recently split `"classification"` to `"binary_classification"` and `"multiclass_classification"`. You can also solve this by installing PyG from master.

**Reply by yousha806** [reply]
Thank you for your help!
